spec/features/collection_drafts/forms/data_identification_form_spec.rb

```diff
@@ -1,11 +1,8 @@
 describe 'Data identification form', js: true do
   before do
     login
-    draft = create(:collection_draft, user: User.where(urs_uid: 'testuser').first)
-    visit collection_draft_path(draft)
-    within '.metadata' do
-      click_on 'Data Identification'
-    end
+    draft = create(:collection_draft, user: User.where(urs_uid: 'testuser').first, entry_title: 'Empty Draft for Data Identificaiton form')
```

Comment:
Typo in Identification.

Reply:
Good catch. I added it to try and find in the test logs. But to use it properly for this and other tests, I would have had to make a few adjustments to the factory, so I've removed it for now.